spk/duplicity/Makefile

```diff
@@ -4,7 +4,8 @@ SPK_REV = 1
 SPK_ICON = src/duplicity.png
 
 BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel
-BUILD_DEPENDS += cross/boto cross/netifaces cross/wrapt cross/$(SPK_NAME)
+BUILD_DEPENDS += cross/boto cross/msgpack-python cross/netifaces cross/rackspace-novaclient
```

Comment:
`cross/msgpack-python` seems to be a run-time dependency (according to the requirements file, for example). If so, then this probably isn't going to work: `cross/msgpack-python` is compiled as a python module for inclusion in the python package, whereas you'd need to create a wheel to be able to include it in the duplicity package.